stubs/unit/aggregated.api.json

```diff
@@ -35,1666 +35,1671 @@
     }
   },
   "case-types": [
```

Comment:
it could be possible to have an example of DateTime in the aggregated api? ignore me if it's there, I couldn't find it